grails-app/conf/RoleFilters.groovy

```diff
@@ -86,13 +95,20 @@ class RoleFilters {
     }
 
     static Boolean needManager(controllerName, actionName) {
-        changeActions.any {
+        def changeActions = changeActions.any {
             actionName?.startsWith(it)
-        } || controllerName?.contains("Workflow") || changeControllers?.contains(controllerName) || managerActions[controllerName]?.contains(actionName)
+        }
+        def controller = controllerName?.contains("Workflow")
+        def changeController = changeControllers?.contains(controllerName)
+        def manager = managerActions[controllerName]?.contains(actionName)
```

Comment:
Should be named as `isManagerAction` (or sth similar).

Reply:
ok